dsl_parser/parser.py

```diff
@@ -70,7 +73,21 @@ def _parse(dsl_string,
            resolver=None,
            validate_version=True,
            validate_intrinsic_function=True,
-           additional_resource_sources=()):
+           additional_resource_sources=(),
+           validate_input_defaults=True):
+    """
+
+    :param dsl_string: DSL string.
+    :param resources_base_path: resources base path.
+    :param dsl_location: DSL location.
+    :param resolver: resolver.
+    :param validate_version: whether to validate version.
+    :param validate_intrinsic_function: whether to validate intrinsic
+        functions.
+    :param additional_resource_sources: additional resource sources.
+    :param validate_input_defaults: whether to validate given input constraints
+        on input default values.
```

Comment:
:param dsl_string: parsing input in form of a string.
    :param resources_base_path: resources base path.
    :param dsl_location: yaml location.
    :param resolver: resolver.
    :param validate_version: whether to validate version of the DSL.
    :param validate_intrinsic_function: whether to validate intrinsic
        functions.
    :param additional_resource_sources: additional resource sources.
    :param validate_input_defaults: whether to validate given input constraints
        on input default values.